behaviour/graql/language/match.feature

```diff
@@ -24,7 +24,65 @@ Feature: Graql Match Clause
     Given transaction is initialised
     Given the integrity is validated
 
-  Scenario: Disjunctions return the union of composing query statements
+  Scenario: `type` matches only the specified type, and does not match subtypes
+
+  Scenario: `sub` matches the specified type and all its subtypes
+
+  # TODO: ????
+  Scenario: `sub!` matches ??? (?)
```

Comment:
what's this?

Reply:
Updated to: `sub! matches the specified type and its direct subtypes`